chainer/link.py

```diff
@@ -197,6 +197,15 @@ def __init__(self):
         finally:
             self._within_init_scope = old_flag
 
+    def __call__(self, *args, **kwargs):
+        try:
+            forward = self.forward
+        except AttributeError:
+            raise TypeError(
+                '{} object has neither a \'Link.__call__\' method overridden'
+                'nor a forward method defined.'.format(self))
```

Comment:
Both articles `a`  are not needed.
"forward" should be surrounded by `''` like `Link.__call__`.

Reply:
Oooops thanks 😉